src/clinical_etl/CSVConvert.py

```diff
@@ -643,6 +646,10 @@ def csv_convert(input_path, manifest_file, minify=False, index_output=False, ver
     if mappings.IDENTIFIER_FIELD is None:
         sys.exit("Need to specify what the main identifier column name is as 'identifier' in the manifest file, "
                  "see README for more details.")
+    mappings.DATE_FORMAT = manifest["date_format"]
+    if mappings.IDENTIFIER_FIELD is None:
```

Comment:
Should this be mappings.DATE_FORMAT?
```suggestion
    if mappings.DATE_FORMAT is None:
```

Reply:
Yes, thank you